.github/actions/cmake-test/action.yml

```diff
@@ -36,6 +35,15 @@ runs:
         # along in the same manner to those test projects via the command line.
         BOOST_ROOT: ${{ steps.install-boost.outputs.BOOST_ROOT }}
         OPENSSL_ROOT_DIR: ${{ steps.install-openssl.outputs.OPENSSL_ROOT_DIR }}
+        CMAKE_INSTALL_PREFIX: ../LAUNCHDARKLY_INSTALL
```

Comment:
I'm arbitrarily installing into this folder here. Later, this folder is injected into the cmake configure step so `find_package` looks there for `launchdarklyConfig.cmake`.

That's one use-case, but it would also be good to check:
1. If you don't give it an install prefix, and just run `cmake --install .`
2. Then if you invoke `find_package` without specifying a custom `CMAKE_PREFIX_PATH`
3. It will work, due to `find_package`'s lookup logic magic.